test/unit-test-helpers/message_bus_helper.js

```diff
@@ -0,0 +1,13 @@
+(function(OO,_) {
+
+  _.extend(OO.MessageBus.prototype,  {
+    published: function(event) {
```

Comment:
This looks to be a little behind the internal master-v4 message_bus_helper file
I can send you the link offline